SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -530,6 +530,31 @@ FRPCErrorInfo SpatialRPCService::ApplyRPC(const FPendingRPCParams& Params)
 	return ApplyRPCInternal(TargetObject, Function, Params);
 }
 
+struct NetWriteFenceResolutionHandler
```

Comment:
nit - I would place this in a private namespace, to avoid it leaking into other parts of the code in unity builds.

Reply:
So if we want to tackle the issue of anonymous namespaces in unity build, I would rather we skip the awkward phase where we drop all anonymous namespace and one has to figure out a local "private" namespace (which is not so private, and might have to be updated when moving code around), and skip to something like what we were using at my previous place, some form of macro implemented like that : https://github.com/improbableio/UnrealEngine/commit/8c606b3f36d05ad9eaa6604aee04917d5aac795a

Then you can write your anonymous namespace like : 

```
namespace UNITY_ANONYMOUS_NAMESPACE
{
  struct Bar
  {};
}

void Foo()
{
  USING_UNITY_ANONYMOUS_NAMESPACE
  Bar Baz;
}
```

The various macros get replaced by a pseudo unique name for each file in unity builds, and empty string outside.
So you can keep true anonymous namespaces outside of unity builds.